packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.tsx

```diff
@@ -16,6 +16,10 @@ export interface IGroupHeaderState {
 }
 
 export class GroupHeader extends BaseComponent<IGroupDividerProps, IGroupHeaderState> {
+  public static defaultProps: IGroupDividerProps = {
+    expandCollapseButtonProps: { 'aria-label': 'expand collapse group' }
```

Comment:
Hmm, since I don't know for sure, is our de facto standard to have default English values instead of none?

Reply:
Just spoke with @micahgodbolt . I recently made a [change in SearchBox](https://github.com/OfficeDev/office-ui-fabric-react/pull/5180) that also uses a default English value for its ariaLabel. It seems that it's okay for now and in the future we'll move towards a language file.

Reply:
We don't want default values since we don't support localization in this repo. We'll need to take it as a prop to the component.

Reply:
I'm talking to david about the need for moving these strings to a resjson file. Having easy to opt into default strings will fix a lot of accessibility issues, but I agree that our final answer shouldn't be putting those English strings into props.

@ThomasMichon any objection to merging this as is and adding this string to the list needed for the resjson file?

Reply:
I also tended to think we shouldn't have any localization in Fabric, but when I see PRs like Coachmark embedding custom accessibility navigation behavior that requires some amount of instruction to the user I think some kind of default is a good idea.